src/Core/Scheduler/Scheduler.js

```diff
@@ -171,10 +245,27 @@ Scheduler.prototype.addProtocolProvider = function addProtocolProvider(protocol,
     this.providers[protocol] = provider;
 };
 
+/**
+ * Get a specific {@link Provider} given a particular protocol.
+ *
+ * @param {string} protocol
+ *
+ * @return {Provider}
+ */
 Scheduler.prototype.getProtocolProvider = function getProtocolProvider(protocol) {
     return this.providers[protocol];
 };
 
+/**
+ * Get a copy of the full list of [providers]{@link Provider} added to the
+ * Scheduler.
+ *
+ * @return {Array<Provider>}
+ */
+Scheduler.prototype.getProviders = function getProviders() {
```

Comment:
This function is unused and I don't see how it could be useful. Maybe you could remove it completely?

Reply:
Yes I agree !